vignettes/r2dii-analysis.Rmd

```diff
@@ -6,17 +6,13 @@ title: "Introduction to r2dii.analysis"
 knitr::opts_chunk$set(
   collapse = TRUE,
   comment = "#>",
-  fig.path = "man/figures/README-",
   out.width = "100%"
 )
 ```
 
 ## Load your r2dii libraries
 
-The first step in your analysis will be to load in the recommended r2dii packages 
-into your current R session. [r2dii.data](https://2degreesinvesting.github.io/r2dii.data) 
-includes fake data to help demonstrate the tool and 
-[r2dii.match](https://2degreesinvesting.github.io/r2dii.match) provides functions 
+The first step in your analysis will be to load in the recommended r2dii packages into your current R session. [r2dii.data](https://2degreesinvesting.github.io/r2dii.data) includes fake data to help demonstrate the tool and [r2dii.match](https://2degreesinvesting.github.io/r2dii.match) provides functions 
```

Comment:
I removed the line breaks. In .Rmd files they are needless and make it difficult to reflow a paragraph after adding or removing some text.